### DISCUSSION TITLE
Unable to create Microsoft Azure account😒

### DISCUSSION THREAD

**Discussion by codewithgulam** [github/india#781] [Students]
#712 

error is : Check the info you entered. It doesn't match the info for this card.

**Comment by Impavan007** [comment]
Same problem tell me if you will find any the solution

**Reply by codewithgulam** [reply]
I think we are out of this internship.Did not get the solution.

**Reply by Impavan007** [reply]
Do you know how to acess azure pass service

**Reply by codewithgulam** [reply]
> Do you know how to acess azure pass service

no, I'm sorry(- _ -) no one here is to help us why it is called community?

**Reply by ey-mentor1** [reply]
You have claim student developer pack.
If that's taking time, I would suggest to create free azure account
